manifests/2.2.1/opensearch-2.2.1.yml

```diff
@@ -30,3 +30,9 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: security
+    repository: https://github.com/opensearch-project/security.git
+    ref: '2.2'
+    platforms:
+      - linux
+
```

Comment:
Seems like you accidentally add an empty line.

Reply:
Thanks @zelinh just removed the space.